core/common/modules/connect/apps/base-app.php

```diff
@@ -183,14 +184,17 @@ public function action_get_token() {
 			$this->redirect_to_admin_page();
 		}
 
-		if ( empty( $_REQUEST['state'] ) || $_REQUEST['state'] !== $this->get( 'state' ) ) {
+		//phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Nonce is verified in line 200.
+		$state = Utils::get_super_global_value( $_REQUEST, 'state' );
+
+		if ( empty( $state ) || $state !== $this->get( 'state' ) ) {
 			$this->add_notice( 'Get Token: Invalid Request.', 'error' );
 			$this->redirect_to_admin_page();
 		}
 
 		$response = $this->request( 'get_token', [
 			'grant_type' => 'authorization_code',
-			'code' => $_REQUEST['code'],
+			'code' => Utils::get_super_global_value( $_REQUEST, 'code' ), //phpcs:ignore WordPress.Security.NonceVerification.Recommended -- Nonce is verified in line 200.
```

Comment:
line 200?

Reply:
Auto complete, didn't notice that.
Removed.